### ISSUE TITLE
Change readme to rst format

### ISSUE THREAD

**Issue by shuttle1987** [oadams/persephone#65]
Unfortunately markdown isn't handled over on PyPi. So this means the page https://pypi.python.org/pypi/persephone/0.1.7 has not got great rendering. Perhaps change to rst format?

**Comment by shuttle1987** [comment]
What happened herehttps://github.com/oadams/persephone/commit/2170239e7437237f3cda3938f49598112e9ceacc ?

**Comment by shuttle1987** [comment]
So I see that there's a README.rst and a README.md at the same time now in recent commits, if we delete README.md we can just close this.

**Comment by oadams** [comment]
I tried that a couple weeks ago but then Github didn't render the readme properly.

**Comment by shuttle1987** [comment]
Oh weird. I haven't personally run into that issue myself.

**Comment by oadams** [comment]
Yeah, as far as I can see Github it doesn't support rst. So the ghetto solution is what I have done: add a step in the release checklist #37 to just use pandoc to convert the markdown into rst. This will be seen in the next release on pypi.

**Comment by shuttle1987** [comment]
GitHub does support rst but if there's errors rst can silently fail to render: https://github.com/github/markup/issues/503

I've created a PR #92 with the necessary fixes.
